packages/api-server/app.js

```diff
@@ -88,14 +88,12 @@ function hasMethod(body, name) {
 }
 
 function getIp(req) {
-  if (
-    Array.isArray(req.headers["x-forwarded-for"]) &&
-    req.headers["x-forwarded-for"].length > 0
-  ) {
-    return req.headers["x-forwarded-for"][0];
+  let ip;
```

Comment:
how about `return (req.headers["x-forwarded-for"] || "").split(",")[0].trim() || req.socket.remoteAddress;`